infrastructure/ai/providers/openai_provider.py

```
@@ -85,8 +80,16 @@ async def _generate_via_chat(self, prompt: Prompt, config: GenerationConfig) ->
         """Chat Completions API 非流式生成"""
         messages = self._build_messages(prompt)
         request_kwargs = self._build_chat_request_kwargs(messages, config)
+        print(f"[LLM] 准备调用 Chat API，模型: {request_kwargs.get('model')}，max_tokens: {request_kwargs.get('max_tokens')}")
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)